.github/workflows/automerge.yml

```diff
@@ -1,6 +1,7 @@
 name: AutoMerge
 on:
   pull_request:
+    types: [opened, reopened, labeled, synchronize]
```

Comment:
Can you add a comment linking to the relevant GitHub docs?

Reply:
```suggestion
    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
    types: [opened, reopened, labeled, synchronize]
```